miv/visualization/connectivity_plot.py

```diff
@@ -0,0 +1,123 @@
+import os
+
+import graphviz
+import matplotlib.pyplot as plt
+import numpy as np
+from pyvis.network import Network
+
+from miv.typing import SpikestampsType
+
+
+def plot_connectivity(
```

Comment:
Please split 'interactive plot' and 'non-interactive plot'.

`plot_connectivity_interactive()` and `plot_connectivity()` for now.